packages.config

```diff
@@ -2,4 +2,5 @@
 <packages>
   <package id="jQuery" version="3.1.1" targetFramework="net46" />
   <package id="NLog" version="4.3.10" targetFramework="net46" />
-</packages>
\ No newline at end of file
+  <package id="LibGit2Sharp" version="0.20.0" targetFramework="net46" />
```

Comment:
;) [Nexus Lifecycle](https://www.sonatype.com/product-nexus-lifecycle) has found that this version of `LibGit2Sharp` violates your company's policies.

Lifecycle recommends using version [0.22.0](https://www.nuget.org/packages/LibGit2Sharp/0.22.0) instead as it does not violate any policies.